astronomy-pipeline/astronomy_transform_functions.py

```diff
@@ -0,0 +1,85 @@
+"""Functions for the astronomy API transform script."""
+
+import json
+import pandas as pd
+
+from astronomy_extract_functions import get_db_connection
+
+
+def load_from_file(filename: str) -> dict:
+    """Load the stories from a file called stories.json"""
+
+    with open(filename, "r", encoding="UTF-8") as f_obj:
+        return json.load(f_obj)
+
+
+def get_data_into_database(raw_data: dict) -> pd.DataFrame:
+    """Converts a list of body position dictionaries into a dataframe."""
+
+    regions = list(raw_data["body_positions"].keys())
+    times = list(raw_data["body_positions"]["1"].keys())
+
+    df_list = []
+    for r in regions:
+        for t in times:
+            time_list = raw_data["body_positions"][r][t]
+            df = pd.DataFrame(time_list)
+            df["region_id"] = int(r)
+            df_list.append(df)
```

Comment:
this could be a list comprehension

Reply:
Either loops or list comprehensions are fine, though I prefer this implementation for nested for-loops.